web/sites/default/config/core.entity_form_display.group.elected_official.default.yml

```diff
@@ -184,6 +176,27 @@ third_party_settings:
         open: false
         description: ''
         required_fields: true
+    group_council_clerk_only:
+      children:
+        - field_abbreviated_name
+        - field_active_elected_official
+        - field_voting_council_member
+        - field_commissioner_number
+        - field_council_district
+        - field_enable_body_content
```

Comment:
The Enable Custom Menu field should be in the Administrative Fields group since it's limited to site admins.